index.html

```diff
@@ -251,11 +265,12 @@ <h4 class="heading-langS">Skills</h4>
                                                      <li><a href="https://github.com/"><img src="resources\github (1).svg" alt="GitHub icon" /></a></li>
                                                      <li><a href="https://www.angellist.com/"><img src="resources\angellist.svg" alt="Angelist icon" /></a></li>
                                   </ul>
+                                                </div>
                                 </section>
                                               </div>
                                               <div class="imgframe">
                                 <img src="./resources/Frame.svg" alt="bottomtri">
-                                </div>
+                                              </div>
                               </footer>
     </main>
```

Comment:
[OPTIONAL]
- Kindly note that this closing `footer` tag has no opening tag, kindly fix it.